Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pmtiles"
-version = "0.15.0"
+version = "0.14.0"
```

Comment:
🤔 was this intentional?

Reply:
yes, this is how release-plz framework publishes - if the value differs from what's released, it will release right away. We only want it to release when we merge the release-pr.